watch-parties/src/test/java/com/google/sps/servlets/VideoSyncServletTest.java

```diff
@@ -29,35 +29,54 @@
 /** Class to test VideoSyncServlet. */
 @RunWith(JUnit4.class)
 public final class VideoSyncServletTest {
-  private HttpServletRequest mockRequest;       
-  private HttpServletResponse mockResponse; 
-
-  @Before
-  public void setUp() {
-    mockRequest = mock(HttpServletRequest.class);       
-    mockResponse = mock(HttpServletResponse.class);   
-  }
-
-  @Test
-  public void testDoGet() throws IOException {
-    StringWriter stringWriter = new StringWriter();
-    PrintWriter writer = new PrintWriter(stringWriter);
-    when(mockResponse.getWriter()).thenReturn(writer);
-
-    VideoSyncServlet servlet = new VideoSyncServlet();
-    servlet.videoStatus = 200;
-    servlet.doGet(mockRequest, mockResponse);
-
-    writer.flush();
-    Assert.assertTrue(stringWriter.toString().contains("200"));
-  }
-
-  @Test
-  public void testDoPost() throws IOException {
-    when(mockRequest.getParameter("status")).thenReturn("100");
-
-    VideoSyncServlet servlet = new VideoSyncServlet();
-    servlet.doPost(mockRequest, mockResponse);
-    Assert.assertEquals(100, servlet.videoStatus); 
-  }
+    private HttpServletRequest HostMockRequest;       
+    private HttpServletResponse HostMockResponse; 
```

Comment:
This value doesn't seem necessary. Could we just share on mock response between all tests?